tests/acceptance/root-test.js

```diff
@@ -79,6 +79,30 @@ test('posting a form', function(assert) {
     assert.equal(data.name, 'World');
   });
 });
+test('posting an array buffer', function(assert) {
```

Comment:
Wrap this test in an `if (typeof ArrayBuffer !== ‘undefined’) {`, which will get the tests passing on phantom (by not running this one) and on chrome...